src/components/MessagesList/MessagesList.vue

```diff
@@ -576,19 +575,20 @@ export default {
 		 */
 		generateDateSeparator(dateTimestamp) {
 			const date = moment.unix(dateTimestamp).startOf('day')
+			const diffDays = moment().startOf('day').diff(date, 'days')
 			// <Today>, <March 18th, 2024>
-			// Relative date is only shown until a week ago
-			if (moment().startOf('day').diff(date, 'days') <= 7) {
+			// Relative date is only shown up to a week ago (inclusive)
+			if (diffDays <= 7) {
 				return t('spreed', '{relativeDate}, {absoluteDate}', {
```

Comment:
Didn't you want to do:
```suggestion
				// TRANSLATORS: <Today>, <March 18th, 2024>
				return t('spreed', '{relativeDate}, {absoluteDate}', {
```